src/constants/index.js

```diff
@@ -59,6 +59,8 @@ import { DiCss3, DiRuby } from "react-icons/di";
 
 export const resumeLink = "https://drive.google.com/file/d/1wkPu9i_e2uAdHmbqe0hEM9BfBheJfxkT/view?usp=sharing";
 
+export const callToAction = 'https://www.linkedin.com/in/mittal-parth';
```

Comment:
@asamaiya00 just for consistency
```suggestion
export const callToAction = "https://www.linkedin.com/in/mittal-parth";
```